tools/mascot/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+           http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <parent>
+    <groupId>dk.alexandra.fresco</groupId>
+    <artifactId>tools-master-pom</artifactId>
+    <version>0.2-SNAPSHOT</version>
+  </parent>
+
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>dk.alexandra.fresco</groupId>
+  <artifactId>mascot</artifactId>
+  <packaging>jar</packaging>
+  <name>mascot</name>
+
+  <dependencies>
+    
+    <dependency>
+      <groupId>dk.alexandra.fresco</groupId>
+      <artifactId>core</artifactId>
+      <version>${fresco-version}</version>
+    </dependency>
+    
+    <dependency>
+      <groupId>dk.alexandra.fresco</groupId>
+      <artifactId>core</artifactId>
+      <version>${fresco-version}</version>
+      <type>test-jar</type>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+    	<groupId>dk.alexandra.fresco</groupId>
+    	<artifactId>ot</artifactId>
+    	<version>${fresco-version}</version>
+    </dependency>
+  </dependencies>
+  
+  <build>
+    <plugins>
+      <plugin>
+    <groupId>org.apache.maven.plugins</groupId>
+    <artifactId>maven-compiler-plugin</artifactId>
+    <version>2.0.2</version>
+    <configuration>
+      <encoding>UTF-8</encoding>
+      <source>1.8</source>
+      <target>1.8</target>
+    </configuration>
+      </plugin>
+      <plugin>
+    <artifactId>maven-assembly-plugin</artifactId>
+    <executions>
+      <execution>
+        <configuration>
+          <appendAssemblyId>false</appendAssemblyId>
+          <archive>
+        <manifest>
+          <mainClass>dk.alexandra.fresco.tools.mascot.Mascot</mainClass>
+        </manifest>
+          </archive>
+          <descriptorRefs>
+        <descriptorRef>jar-with-dependencies</descriptorRef>
+          </descriptorRefs>
+          <finalName>fresco-tools-mascot</finalName>
```

Comment:
Do we package mascot - why?

Reply:
Whoops that was a copy and paste. Gone now.